tutorials/introductory/getting_started.py

```diff
@@ -0,0 +1,842 @@
+# outer edge.
+budget_pie = ax.pie(budget,
+                    colors=colors,
+                    labels=descriptions,
+                    explode=explode,
+                    autopct=partial(autopct_format, group=budget),
+                    startangle=45,
+                    pctdistance=0.85,
+                    labeldistance=1.125,
+                    wedgeprops=dict(width=0.3),
+                    shadow=True)
+
+# The pie() method unpacks into three Artist objects. The Artists wedges,
+# texts, and autotexts have their own methods for addtional customization.
+wedges, texts, autotexts = budget_pie
+
+# The unpacked wedges variable serve as handles for legend.
+# Info from categories correspond to respective wedge instead of redundant
+# labeling from previous pie chart.
+# Legend has title keyword argument.
+# Keyword argument bbox_to_anchor with loc places legend at specific point.
+# Tuple floats are coordinates for Figure as row and column of Axes.
+# Keyword argument loc works with bbox_to_anchor to determine part of legend
+# for placement. Without bbox_to_anchor, Matplotlib automatically manages
+# coordinates in relation to parameters of loc.
+ax.legend(wedges,
+          categories,
+          title='Categories',
+          bbox_to_anchor=(0.125, 0.5),
+          loc='center right')
+
+ax.set_title('Average Monthly Income Expenses')
+ax.axis('equal')
+
+# The Figure method tight_layout() manages space between all Artists to
+# maximize visiblity on Figure. This method also contains various
+# parameters for configuration.
+fig.tight_layout()
+
+plt.show()
+
+##############################################################################
+#
+# .. note::
+#
+#   The ``partial`` function in functools works as a callable for simplifying
+#   a function's arguments. In the ``autopct`` keyword argument, only one
+#   argument is provided, the data acting as a percentage equivalent. The
+#   ``autopct_format`` function requires two arguments, so ``partial`` takes
+#   the argument for ``group`` and sets it to ``budget``. This smaller
+#   signature object then behaves as the same function with one fewer argument.
+#   For details about the functools module, see
+#   `functools
+#   <https://docs.python.org/3/library/functools.html#module-functools>`_.
+#
+# Multiple Graphs within a Figure
+# -------------------------------
+#
+# For multiple graphs using a single Figure, explicit and implicit approaches
+# use a similar convention for mapping out multiple Axes. Matplotlib manages
+# more than one Axes in a two-dimensional matrix. They are arranged by row
+# amount and then by column amount.
+#
+# Implicit coding uses a separate method with a similar name. The method
+# ``plt.subplot`` also includes a third argument to represent the specific
+# Axes involved.
+#
+# When looking for more complex solutions to multiple graphs within a Figure,
+# use the :class:`matplotlib.gridspec.GridSpec` module to organize the layout.
```

Comment:
It may be better to call out `SubFigure` instead now.

Reply:
I think adding a reference to SubFigure here isn't too necessary at the moment for an introductory user. This can always be added later if appropriate.